### ISSUE TITLE
Hide comments from specific user

### ISSUE THREAD

**Issue by malamud** [umputun/remark#291]
As a user I would prefer not seeing comments from certain users.

Do I want to see answers on hidden user’s comments? No, I don’t.  
Do I need a possibility to make that user visible again? No, I don’t.

**Comment by umputun** [comment]
This is not a bad idea, but I think such action should be reversible, i.e. "make that user visible again" better to be supported. Another thing to keep in mind - this functionality may require partial hiding of some subtrees.

As of now, we don't have any support for lists of muted users and no way to attach it to other users. I think, implementing backend side for this should be very doable and even not too hard. Not sure about UI side of this.

Another idea - maybe all of this can be done completely on UI side? @Reeywhaar - what do you think? Maybe even the list of such user can be stored locally in the browser and UI will do the actual hiding. I'm not sure how easy this to make on the browser side but doesn't sound like a big deal to me. Or, maybe an in-between solution with server-side storage API and hiding on UI side.

**Comment by malamud** [comment]
I think it can be done completely on UI side. The list of muted users can be placed in browser storage or in cookies thereby muted user can be made visible by clearing storage or cookies. Maybe it’s not an elegant solution but the simplest one. Nevertheless if it possible to make list of muted user as part of UI it would be nice.

**Comment by Reeywhaar** [comment]
Nothing complex here. Client only way is to store in localStorage.

Additionaly few endpoints can be added, so `/api/v1/user` will return additional list of blocked user ids, so list can be shared across multiple devices. Also, I haven't looked at #279 feature yet, what response `/api/v1/user` has this way.

**Comment by umputun** [comment]
I don't think this has to support cross-device functionality. The reason for such "local filtration" can be not just "I hate this guy", but also "he's writing too many and too wordy comments  I don't want to see on iphone, but fine on mac". 

I suggest making it frontend-only function without any API changes, at least in the short term.
